api/get/docker_to_ecr.sh

```diff
@@ -1,4 +1,4 @@
 aws ecr get-login-password --region eu-west-2 | docker login --username AWS --password-stdin 605126261673.dkr.ecr.eu-west-2.amazonaws.com
-docker tag 3c5f71e357e4 605126261673.dkr.ecr.eu-west-2.amazonaws.com/three-musketeers:api-get
+docker tag 41aa60b2037d 605126261673.dkr.ecr.eu-west-2.amazonaws.com/three-musketeers:api-get
```

Comment:
No need to change the id every time. Assign a tag when building e.g. `docker build -create_report .` and this can be used when doing `docker tag` for the ecr

Reply:
😱